pages/whats-new/2021-03-09-grouparoo-generate.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Code Config Generators"
+date: "2021-03-09"
+tags: ["new"]
+description: "Grouparoo's Declarative Sync feature revolves around the use of the generate command within our CLI tool."
+image: "declarative-data-sync/declarative-sync.png"
+---
+
+Last month we launched an exciting new feature, [Declarative Sync](/blog/declarative-data-sync). This feature revolves around the use of [our CLI tool](/docs/cli), specifically [the `generate` command](/docs/cli/config#generate).
+
+The generate command uses your input to write a configuration file to your file system. You can then use this file to configure the various objects in Grouparoo, like [Apps](/docs/config/apps/community), [Sources](/docs/config/sources/community), or [Destinations](/docs/config/destinations/community).
+
+These templates are then read and processed by Grouparoo when the server starts. This makes it trivial to import and export data from your various data sources and destinations without having to click around the UI.
```

Comment:
[idea] I don't think that `without having to click around the UI` is the alternative, but perhaps: 
> This makes it trivial to import and export data from your various data sources and destinations by just writing a few lines of JSON.